apps/code-examples/src/app/code-examples/forms/selection-box/checkbox/demo.component.ts

```diff
@@ -10,7 +10,7 @@ import {
 } from '@angular/forms';
 import { SkyIdModule } from '@skyux/core';
 import { SkyCheckboxModule, SkySelectionBoxModule } from '@skyux/forms';
-import { SkyIconModule } from '@skyux/indicators';
+import { SkyIconModule } from '@skyux/icon';
```

Comment:
Code examples (i.e. our consumers) shouldn't be referencing `@skyux/icon`, correct? These should probably remain as-is until SKY UX 11.